site/viamguides/src/plant-watering-device-workshop/plant-watering-device-workshop.md

```diff
@@ -103,98 +151,75 @@ Now you are done!
 
 Make sure you understand these concepts before moving ahead.
 
-## Configure your Plant Watering Device
+## Configure your machine
+Duration: 10
 
-Duration: 5
+1. In [the Viam app](https://app.viam.com/fleet/dashboard) under the **LOCATIONS** tab, create a machine by typing in a name and clicking **Add machine**.
+   ![add machine](assets/addMachine.png)
+1. Click **View setup instructions**.
+   ![setup instructions](assets/awaiting.png)
+1. To install `viam-server` on your Raspberry Pi (so you can communicate with and control your sensor and the RGB LED), select the `Linux / Aarch64` platform for the Raspberry Pi, and leave your installation method as [`viam-agent`](https://docs.viam.com/how-tos/provision-setup/#install-viam-agent).
+   ![select platform](assets/selectPlatform.png)
+1. Use the `viam-agent` to download and install `viam-server` on your Raspberry Pi. Follow the instructions to run the command provided in the setup instructions from the SSH prompt of your Raspberry Pi.
+1. The setup page will indicate when the machine is successfully connected.
+    <img src="assets/machineConnected.png" alt="machine connected" width="450"/>
 
-- Go to your Viam app account and [sign in](https://app.viam.com/fleet/locations/).
+With a machine configured and connected, it's time to add the peripherals.
 
-- Under the **LOCATIONS** tab, create a new machine and follow the setup steps.
+<!-- ------------------------ -->
 
-- Configure your components, start with the board: selecting `pi` for the Raspberry Pi 4 or `pi5` for the Raspberry Pi 5.
+## Configure your Raspberry Pi
+Duration: 5
 
-![Board Configuration](assets/board.png)
+1. In [the Viam app](https://app.viam.com/fleet/locations), find the **CONFIGURE** tab.
+1. Click the **+** icon in the left-hand menu and select **Component**.
+  ![Board Configuration](assets/addBoard.png)
+1. Select `board`, and find the `raspberry-pi:rpi4` (or `raspberry-pi:rpi5` for the Raspberry Pi 5) module. Click **Add Module**. Leave the default name `board-1` for now, then click **Create**. This adds the module for working with the Raspberry Pi's GPIO pins.
```

Comment:
If we're noting the potential use of a Raspberry Pi 5, we should include that option in the "Install Raspberry Pi OS" step where we say:

> Click **CHOOSE DEVICE**. Select your model of Pi, which is Raspberry Pi 4.

Possible update could be:

> Click **CHOOSE DEVICE**. Select your model of Pi, i.e. Raspberry Pi 4 or Raspberry Pi 5.